File: NumGuess/game.cpp
#include <iostream>
#include <cstdlib>

using namespace std;

void play(int minNumber, int maxNumber, int attempt)
{
    // Providing a seed value
    srand((unsigned) time(NULL));

    //rownanie, ktore losuje liczbe z przedzialu minNum - maxNum
    int random = minNumber + (rand() % maxNumber);
    int choice = 0;

    cout << "Choose number from " << minNumber << " to " << maxNumber << endl << endl;

    for(int i = 0 ; i < attempt ; i++)
    {
        if(i == (attempt - 1)) cout << "This is Your last chance!" << endl;
        cout << "Choose Youe number: " ;
        cin >> choice;

        if(choice == random)
        {
            cout << endl << "Bravo, You got this!"<<endl;
            cout << "That was Your " << i+1 << " shoot!" << endl;
            break;
        } else if(choice > random){
            cout << "Too much!" << endl << endl;
        } else if(choice < random){
            cout << "Too few!" << endl << endl;
        }

    }

}

File: NumGuess/game.py
import random

def game(min_num, max_num, attempt):
    random_num = random.randint(min_num, max_num)

    for x in range(0, attempt):
        guess = input('Guess Number!')
        guess = int(guess)

        if(guess == random_num):
            print('Bravo, this is number which You had to guess!')
            break
        elif(guess > random_num):
            print('To much')
        else:
            print('Not enough!')

        if(x == (attempt - 2)):
            print('This is Your last chance!')
        